source/howto/SnapshotVM.rst

```diff
@@ -1,5 +1,183 @@
-=================================
-Snapshot a VM to create an image
-=================================
-
-Documentations coming soon
+**********************
+Instance Snapshot

```

Comment:
This title isnt obvious what the document is for, the previous title was more informative